### PR TITLE
Update styling for show more updates

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Update styling to match device page
image
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/dashboard/ha-config-updates.ts [hunk 1]
```diff
@@ -125,6 +125,22 @@ class HaConfigUpdates extends LitElement {
           height: 24px;
           width: 24px;
         }
+        button.show-more {
```
Comment (bramkragten) [inline src/panels/config/dashboard/ha-config-updates.ts:128]: Use <https://github.com/home-assistant/frontend/blob/dev/src/resources/styles.ts#L151-L162>

Comment (ludeeus) [reply src/panels/config/dashboard/ha-config-updates.ts:128]: Then it will no longer match?

Comment (bramkragten) [reply src/panels/config/dashboard/ha-config-updates.ts:128]: Oh that was used? <https://github.com/home-assistant/frontend/pull/11435/files#diff-ae93cc12a13745395838539659cd787e05014b8f4e6fbc262472e7f792e82bceR97>

Comment (bramkragten) [reply src/panels/config/dashboard/ha-config-updates.ts:128]: Then remove that 😛

Comment (ludeeus) [reply src/panels/config/dashboard/ha-config-updates.ts:128]: it was

Comment (ludeeus) [reply src/panels/config/dashboard/ha-config-updates.ts:128]: yeah, did not see the import, will remove shortly

**Comment by spacegaier** [comment]
The spelling is a bit inconsistent. The header says "8 updates" with lower case "u", but the show more link has an upper case "U". I guess the lower case one is more logical.

**Comment by ludeeus** [comment]
Translations are handled on localise ;)

**Comment by spacegaier** [comment]
> Translations are handled on lokalise ;)

But not the initial EN one, since that one is in `src/translations` or did something change?

```json
"more_updates": "+ {count} Updates",
```

**Comment by ludeeus** [comment]
https://github.com/home-assistant/frontend/pull/11437

**Review by bramkragten — APPROVED** [review]

**Pull request merged**